adapters/handlers/rest/handlers_objects.go

```diff
@@ -45,11 +45,11 @@ type ModulesProvider interface {
 type objectsManager interface {
 	AddObject(context.Context, *models.Principal, *models.Object) (*models.Object, error)
 	ValidateObject(context.Context, *models.Principal, *models.Object) error
-	GetObject(context.Context, *models.Principal, strfmt.UUID, additional.Properties) (*models.Object, error)
+	GetObject(_ context.Context, _ *models.Principal, class string, _ strfmt.UUID, _ additional.Properties) (*models.Object, error)
```

Comment:
I think that `_` and `class` argument names can be removed, to be consistent with other method declarations

Reply:
I afraid I disagree with you on this since the arguments with "_" have clear meanings but since the "class" argument is a string it needs to convey its meaning too.